src/models/CourseInstance.js

```diff
@@ -7,7 +7,7 @@ const courseInstanceSchema = new mongoose.Schema(
       required: true,
       ref: "Instructor",
     },
-    students: [
+    student: [
```

Comment:
This should still be plural students right? Since it's an array of students.

Reply:
You are corret. I can alter that, but it will require changes to the seeder files and aggregations. If you would like that to be done let me know and I will take care of it.

Reply:
Yeah, I think this will be a necessary change unfortunately.

Reply:
Undertsood. Do I need to pull from main before altering and pushing?

Reply:
I don't _think_ that'll be needed

Reply:
These changes have been made through all models, seeders, and queries for arrays containing students, studentWorkProjects, and studentOutcomes.